Legion.BackEnd/app/Announcement.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Announcement extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'title', 'content'
+    ];
+
+    /**
+     * The table associated with the model.
+     *
+     * @var string
+     */
+    protected $table = 'announcements';
```

Comment:
You really don't have to comment this.  I think it's pretty obvious what this does lol.  If it's best practices, though, go ahead.

Reply:
Might as well have it for consistency is my reasoning.

Reply:
Alright, at long as it doesn't get out of hand.